From 3de2c5707c1d3139c7005a40dfcdb6469d539dbd Mon Sep 17 00:00:00 2001
From: "carlosgc@webkit.org"
 <carlosgc@webkit.org@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Wed, 5 Oct 2016 09:23:52 +0000
Subject: [PATCH] Unreviewed. Fix GTK+ build with
 USE_REDIRECTED_XCOMPOSITE_WINDOW disabled.

* UIProcess/AcceleratedDrawingAreaProxy.cpp:
(WebKit::AcceleratedDrawingAreaProxy::setNativeSurfaceHandleForCompositing):

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@206806 268f45cc-cd09-0410-ab3c-d52691b4dbfc
---
 Source/WebKit2/UIProcess/AcceleratedDrawingAreaProxy.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebKit2/UIProcess/AcceleratedDrawingAreaProxy.cpp b/Source/WebKit2/UIProcess/AcceleratedDrawingAreaProxy.cpp
index fddb874..31655cb 100644
--- a/Source/WebKit2/UIProcess/AcceleratedDrawingAreaProxy.cpp
+++ b/Source/WebKit2/UIProcess/AcceleratedDrawingAreaProxy.cpp
@@ -270,7 +270,7 @@ void AcceleratedDrawingAreaProxy::setNativeSurfaceHandleForCompositing(uint64_t
         m_pendingNativeSurfaceHandleForCompositing = handle;
         return;
     }
-    m_webPageProxy.process().send(Messages::DrawingArea::SetNativeSurfaceHandleForCompositing(handle), m_webPageProxy.pageID(), IPC::DispatchMessageEvenWhenWaitingForSyncReply);
+    m_webPageProxy.process().send(Messages::DrawingArea::SetNativeSurfaceHandleForCompositing(handle), m_webPageProxy.pageID(), IPC::SendOption::DispatchMessageEvenWhenWaitingForSyncReply);
 }
 
 void AcceleratedDrawingAreaProxy::destroyNativeSurfaceHandleForCompositing()
-- 
2.1.4

